cohorts/survival.py

```diff
@@ -51,13 +53,18 @@ def plot_kmf(df,
     """
     kmf = KaplanMeierFitter()
     if threshold is not None:
-        if threshold == 'median':
+        is_median = threshold == "median"
+        if is_median:
             threshold = df[condition_col].median()
+        label_suffix = "%.2f" % threshold
         condition = df[condition_col] > threshold
-        label = '{} > {:.2f}'.format(condition_col, threshold)
+        label_no_condition = "%s ≤ %s" % (condition_col, label_suffix)
```

Comment:
I prefer `%` since it's just less chars; I imagine everyone prefers what they're used to?